docs/installation/server-config/security.rst

```diff
@@ -0,0 +1,163 @@
+.. _ksql-security:
+
+Securing KSQL
+-------------
+
+To connect to a secured Kafka cluster, Kafka client applications must provide their security credentials. In the same way,
+KSQL is configured so that the KSQL servers are authenticated and authorized, and data communication is encrypted when
+communicating with the Kafka cluster. KSQL can be configured for:
+
+- :ref:`SSL for encryption <kafka_ssl_encryption>`
+- :ref:`SASL for authentication <kafka_sasl_auth>`
+- :ref:`HTTPS for Confluent Schema Registry <schema_registry_http_https>`
+
+-------------------------------------------------
+Using KSQL with an Apache Kafka Secured with ACLs
+-------------------------------------------------
+
+You can use KSQL with Apache Kafka clusters that are secured with ACLs. The behavior depends on whether the cluster is
```

Comment:
> The behavior depends on whether the KSQL servers are configured for interactive use via the KSQL CLI (default) or for non-interactive, headless use.

We should also provide a link where we describe how interactive/non-interactive is being configured.

Reply:
@miguno i couldn't find this information - do you know where this is? i could only find information that describes the value and behavior of interactive queries.

Reply:
I meant a link to the KSQL Configuration section where we describe how to configure KSQL cluster for interactive use (default) vs. non-interactive/headless use (`--queries-file`).  I suppose this part of the KSQL docs is still WIP.

Reply:
i have a added this link: https://github.com/confluentinc/ksql/blob/8cfd40a4de62222923c45a436f50c15c4a8d7582/docs/installation/server-config/index.rst